src/features/duplicates/hooks/useFieldSettings.ts

```diff
@@ -39,5 +39,13 @@ export default function useFieldSettings(duplicates: ZetkinPerson[]) {
     }
   });
 
-  return { fieldValues, initialOverrides };
+  function entryContainsOneValue(entry: Array<string | string[]>) {
+    return entry[1].length < 2;
+  }
+
+  const hasConflictingValues = !Object.entries(fieldValues).every(
+    entryContainsOneValue
+  );
```

Comment:
There's some unnecessary negation going on here, making the code slightly more difficult to read for me than I think it needs to be. I would have "flipped it" around so that instead of making sure that _every_ entry has _one_ value, it would check whether _any_ value has _multiple_ (using `some`, and the opposite logic of `entryContainsOneValue()`).

But that's just a matter of taste! One thing that you've done here which is very good, is to give the variable a name that explains everything. By just reading the name, given that I trust the code (which I will be able to do if I'm reading this after this has been reviewed or merged) I can immediately understand that the purpose of the logic is do figure out whether _there are conflicting values_. That's a huge benefit of naming things nicely! 😊

Reply:
Thanks for the feedback! My aim was to make it readable and understandable. 

I did rewrite it using `some` instead and can see the difference it makes. It can be easy to miss a single `!` when reading it for the first time.